tests/tracking/test_client.py

```diff
@@ -75,28 +70,59 @@ def test_client_create_run_overrides(mock_store):
     )
 
 
-def test_client_search_runs(mock_store, mock_search_filter):
-    experiment_ids = [mock.Mock() for _ in range(5)]
+class SearchFilterMatcher:
+    """Matches a SearchFilter object by comparing it to the given filter string."""
+    def __init__(self, filter):
+        self.filter = filter
 
-    # Test defaults for view type and max results
-    MlflowClient().search_runs(experiment_ids, "metrics.acc > 0.93")
-    mock_store.search_runs.assert_called_once_with(experiment_ids=experiment_ids,
-                                                   search_filter=mock_search_filter,
+    def __eq__(self, other):
+        assert(isinstance(other, SearchFilter))
+        return self.filter == other.filter_string
+
+
+def test_client_search_runs_defaults(mock_store):
```

Comment:
I broke these tests out since it reduced the overall code; we no longer need the comments or the reset_mock.

Reply:
legit!